docs/resources/external_location.md

```diff
@@ -124,6 +124,7 @@ The following arguments are required:
 - `owner` - (Optional) Username/groupname/sp application_id of the external location owner.
 - `comment` - (Optional) User-supplied free-form text.
 - `skip_validation` - (Optional) Suppress validation errors if any & force save the external location
+- `fallback` - (Optional) Indicates whether fallback mode is enabled for this external location. When fallback mode is enabled, the access to the location falls back to cluster credentials if UC credentials are not sufficient.
```

Comment:
What is the default behavior if not specified?

Reply:
https://docs.databricks.com/en/data-governance/unity-catalog/hms-federation/hms-federation-internal.html#fallback - it's not enabled by default

Reply:
Gotcha, could you just add that here? otherwise this LGTM.